packages/@storefront/core/test/unit/services/tracker.ts

```diff
@@ -59,11 +59,23 @@ suite('Tracker Service', ({ expect, spy, stub, itShouldExtendBaseService }) => {
   });
 
   describe('init()', () => {
+    const sessionId = 'foo';
+    const SET_SESSION_ID = 'SET_SESSION_ID';
+
+    beforeEach(() => {
+      app.flux = <any> {
+        on,
+        store: { dispatch: spy() },
```

Comment:
Capture the dispatch-specific `spy` in a variable (ie. `dispatch`); make assertions on `dispatch` directly.

Reply:
Done.